applications/CoSimulationApplication/python_scripts/data_transfer_operators/copy_single_to_dist.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import print_function, absolute_import, division  # makes these scripts backward compatible with python 2.6 and 2.7
+
+# Importing the base class
+from KratosMultiphysics.CoSimulationApplication.base_classes.co_simulation_data_transfer_operator import CoSimulationDataTransferOperator
+
+# Other imports
+
+
+def Create(settings):
+    return CopySingleToDist(settings)
+
+class CopySingleToDist(CoSimulationDataTransferOperator):
+
```

Comment:
Also here completing the name and a description of the functionality of the class would be helpful.